Client/Frontend/Settings/Main/Account/SyncNowSetting.swift

```diff
@@ -239,10 +239,11 @@ class SyncNowSetting: WithAccountSetting {
     }
 
     private func addAccessoryView(with image: UIImageView, to cell: UITableViewCell) {
-        accessoryViewContainer.addArrangedSubview(image)
-        accessoryViewContainer.addArrangedSubview(troubleshootButton)
-
-        cell.accessoryView = accessoryViewContainer
+        DispatchQueue.main.async {
+            self.accessoryViewContainer.addArrangedSubview(image)
+            self.accessoryViewContainer.addArrangedSubview(self.troubleshootButton)
+            cell.accessoryView = self.accessoryViewContainer
```

Comment:
Nit: seems like it would be preferable/safer to use a weak reference here (`[weak self] in`); is there any reason we need a strong reference?